### PR TITLE
Document linguistics-tokens.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

**Review by geirst — APPROVED** [review]
👍

File: en/reference/schema-reference.html [hunk 1]
```diff
@@ -3840,6 +3840,27 @@ <h2 id="summary">summary</h2>
...
 </td>
 </tr>
+<tr><td style="white-space: nowrap">linguistics-tokens</td>
+<td>Zero to one</td>
+<td>
+<p id="linguistics-tokens">
+Make the value returned in results from this summary field be an array
+of the tokens indexed in the source field. Multiple tokens at the same
+location are put into a nested array.
+
```
Comment (kkraune) [inline en/reference/schema-reference.html:3850]:
```suggestion
</p>
<p>
```

**Review by kkraune — APPROVED** [review]
I love this :-)

**Review by geirst — APPROVED** [review]
👍

File: en/text-matching.html [hunk 2]
```diff
@@ -43,49 +43,63 @@ <h2 id="index-and-attribute">Index and attribute</h2>
...
++
+     fieldset default {
+         fields: artist, album
+     }
 </pre>
+</p>
 
-<p>Flush and dump index data (file name can change for subsequent flushes):</p>
+<p>
+Redepoy the application to enable the new document summary:
```
Comment (geirst) [inline en/text-matching.html:73]:
```suggestion
Redeploy the application to enable the new document summary:
```

**Review by geirst — APPROVED** [review]
👍

File: en/text-matching.html [hunk 3]
```diff
@@ -126,55 +126,43 @@ <h2 id="index-and-attribute">Index and attribute</h2>
...
 -            expression: bm25(album) + 0.25 * sum(query(user_profile) * attribute(category_scores))
 +            expression: 0.25 * sum(query(user_profile) * attribute(category_scores))
          }
+     }
+ 
 </pre>
 <p>
-  Run the tutorial again using the new schema, stop efter feeding.
-  Find the snapshot number (here 12) and inspect data from the attributes:
+Run the tutorial again using the new schema, stop efter feeding.
```
Comment (geirst) [inline en/text-matching.html:150]: Typo: efter -> after

**Review by geirst — APPROVED** [review]
👍  The reference doc for the 'tokens' transform must also be updated.

**Review by geirst — APPROVED** [review]
👍

**Comment by geirst** [comment]
This is part of Vespa 8.252.17. Will merge once that release is available open source.

**Pull request merged**